crud/common/sharding/sharding_func.lua

```diff
@@ -43,13 +47,30 @@ end
 local function get_function_from_G(func_name)
     local chunks = string.split(func_name, '.')
     local sharding_func = _G
+    local sharding_module = false
+    local ok
+
+    if sharding_module_names[chunks[1]] then
+        ok, sharding_func = pcall(require, chunks[1])
+        if not ok then
+            return nil
+        end
+
+        sharding_module = true
+        table.remove(chunks, 1)
+    end
 
     -- check is the each chunk an identifier
     for _, chunk in pairs(chunks) do
         if not utils.check_name_isident(chunk) or sharding_func == nil then
             return nil
         end
-        sharding_func = rawget(sharding_func, chunk)
```

Comment:
Shouldn't `rawget` satisfy both cases?

Reply:
Let's add a comment in code so it will be obvious for future generations

Reply:
Done:

https://github.com/tarantool/crud/blob/a479871fd0e793990aab7e0b0ca86ae59277e5ce/crud/common/sharding/sharding_func.lua#L69-L71